src/EditorFeatures/Core/FindUsages/IDefinitionsAndReferencesFactory.cs

```diff
@@ -172,6 +175,25 @@ internal static class DefinitionItemExtensions
             return properties;
         }
 
+        private static ImmutableArray<AdditionalProperty> GetDisplayableProperties(ISymbol definition)
+        {
+            var displayableProperties = ImmutableArray<AdditionalProperty>.Empty;
+
+            var containingType = definition.ContainingType;
+            if (containingType != null)
```

Comment:
I see that we are using `ISymbol.ContainingType.Name` here to get the column value for definition entries, but use the syntax node based approach for reference entries (walk up syntactically to containing type declaration and compute it's name). Should be use uniform approach/API for computing this information for both cases?